docs/integrations/logfire.md

```diff
@@ -0,0 +1,164 @@
+# Logfire
+
+[Logfire](https://docs.pydantic.dev/logfire/), a new tool from Pydantic, is built on OpenTelemetry. As Mirascope is also built on Pydantic, it's appropriate for us to ensure seamless integration with them.
+
+## How to use Logfire with Mirascope
+
+```python
+pip install logfire
+
+from mirascope.integrations.logfire import with_logfire
+```
+
+`with_logfire` is a decorator that can be used on all Mirascope classes to automatically log both Mirascope calls and also all our [supported LLM providers](https://docs.mirascope.io/latest/SUPPORTED_PROVIDERS/).
```

Comment:
update, not just "classes" but also "functions"?

Does it work on tools? tool calls? Might be worth being just a little more specific here

Reply:
updated below helper text to include tools